ra/registration-authority.go

```diff
@@ -216,10 +224,11 @@ func (ra *RegistrationAuthorityImpl) validateContacts(contacts []*core.AcmeURL)
 			continue
 		case "mailto":
 			rtt, err := validateEmail(contact.Opaque, ra.DNSResolver)
-			ra.stats.TimingDuration("RA.DNS.RTT.MX", rtt, 1.0)
+			ra.stats.TimingDuration("RA.DNS.RTT", rtt, 1.0)
```

Comment:
Why remove `.MX`? Currently all the other RTT metrics contain the RR type, not sure it's *entirely* necessary but it might be nice down the line to debug something.